src/btree/bt_slvg.c

```diff
@@ -689,7 +689,7 @@ __slvg_trk_leaf(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk, uint8_t *ad
          * Page flags are 0 because we aren't releasing the memory used to read the page into memory
          * and we don't want page discard to free it.
          */
-        WT_ERR(__wt_page_inmem(session, NULL, dsk, 0, &page, NULL));
+        WT_ERR(__wt_page_inmem(session, NULL, dsk, 0, 0, &page, NULL));
```

Comment:
I think passing `0` here will mean that the code goes from tracking the read size to tracking `0` when called via salvage. I expect that would make salvage cache tracking wrong.

Reply:
Since the `WT_PAGE_DISK_ALLOC` flag is not set here, we would not track the size regardless.